homeassistant/components/unifi/device_tracker.py

```diff
@@ -62,111 +65,146 @@
 })
 
 
-async def async_get_scanner(hass, config):
-    """Set up the Unifi device_tracker."""
-    host = config[DOMAIN].get(CONF_HOST)
-    username = config[DOMAIN].get(CONF_USERNAME)
-    password = config[DOMAIN].get(CONF_PASSWORD)
-    site_id = config[DOMAIN].get(CONF_SITE_ID)
-    port = config[DOMAIN].get(CONF_PORT)
-    verify_ssl = config[DOMAIN].get(CONF_VERIFY_SSL)
-    detection_time = config[DOMAIN].get(CONF_DETECTION_TIME)
-    monitored_conditions = config[DOMAIN].get(CONF_MONITORED_CONDITIONS)
-    ssid_filter = config[DOMAIN].get(CONF_SSID_FILTER)
+async def async_setup_scanner(hass, config, sync_see, discovery_info):
```

Comment:
This is no longer used and can be removed.

Reply:
Used to trigger a config flow import. Works perfectly fine 👍